lib/IDE/CodeCompletion.cpp

```diff
@@ -1722,7 +1722,8 @@ void CodeCompletionCallbacksImpl::doneParsing() {
   }
 
   case CompletionKind::AttributeBegin: {
-    Lookup.getAttributeDeclCompletions(IsInSil, AttTargetDK);
+    bool isGlobalVariable = (AttTarget.hasValue() && AttTarget.getValue() == DeclKind::Var && CurDeclContext.isLocalContext());
```

Comment:
I believe this needs to be `CurDeclContext->isLocalContext()`

Reply:
Oh I forgot it was a pointer, there, fixed it